skl2onnx/operator_converters/GradientBoosting.py

```diff
@@ -22,10 +22,20 @@ def convert_sklearn_gradient_boosting_classifier(scope, operator, container):
 
     if op.n_classes_ == 2:
         transform = 'LOGISTIC'
-        base_values = [op.init_.prior]
+        # class_prior_ was introduced in scikit-learn 0.21.
+        if hasattr(op.init_, 'class_prior_'):
```

Comment:
This could be condensed into:
base_values = op.init_.class_prior_ if hasattr(op.init_, 'class_prior_') else op.init_.priors